avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/FeatureListGeneration.scala

```diff
@@ -300,7 +300,8 @@ object FeatureListGeneration {
                     // the format of each line is feature_entry,count
                     // first get feature_entry, if need process prefix (ntv), remove prefix from feature_entry, make
                     // sure prefix is unique
-                    val lineWithoutCount = line.split(SPLIT_REGEX).head
+                    val words = line.split(SPLIT_REGEX)
+                    val lineWithoutCount = if (words.isEmpty) "" else words.head
```

Comment:
can you also take a look at line 376, might have similar bug

Reply:
okay, I added it and added an unit test for it. Thanks